beta/src/pages/learn/thinking-in-react.md

```diff
@@ -2,17 +2,18 @@
 title: Thinking in React
 ---
 
+
 <Intro>
 
-React can change how you think about the designs you look at and the apps you build. Where once you might have seen a forest, after working with React, you will appreciate the individual trees. React makes it easier to think in design systems and UI states. In this tutorial, we'll guide you through the thought process of building a searchable product data table with React.
+React आपके द्वारा देखे जाने वाले डिज़ाइन और आपके द्वारा बनाए गए ऍप्स के बारे में आपके सोचने के तरीके को बदल सकता है।जहां एक बार आपने जंगल को देखा है, React के साथ काम करने के बाद, आप अलग-अलग पेड़ों की सराहना करेंगे।React डिजाइन प्रणाली और UI states में सोचना आसान बनाता है.इस ट्यूटोरियल में, हम आपको React के साथ खोज योग्य उत्पाद डेटा तालिका बनाने की विचार प्रक्रिया के माध्यम से मार्गदर्शन करेंगे।
 
 </Intro>
 
-## Start with the mockup {/*start-with-the-mockup*/}
+## मॉकअप के साथ शुरू करें {/*स्टार्ट-विद-द-मॉकअप*/}
```

Comment:
`## मॉकअप के साथ शुरू करें {/*स्टार्ट-विद-द-मॉकअप*/}`
In these headings we should not translate the part which is enclosed in curly brackets `{/*स्टार्ट-विद-द-मॉकअप*/}` since that is used for ref links

`## मॉकअप के साथ शुरू करें {/*स्टार्ट-विद-द-मॉकअप*/}` => `## मॉकअप के साथ शुरू करें {/*start-with-the-mockup*/}`